src/App.tsx

```diff
@@ -83,6 +91,33 @@ function App() {
         }
     }
 
+    function animationAddHandler(name: string) {
+        console.log(name);
+        if (animations["animation0"] === "none") {
+            setAnimations(prevState=> (
+                {...prevState, "animation0": name}
+            ));
+        }
+        else if (animations["animation1"] === "none") {
+            setAnimations(prevState=> (
+                {...prevState, "animation1": name}
+            ));
+        }
+        else if (animations["animation2"] === "none") {
+            setAnimations(prevState=> (
+                {...prevState, "animation2": name}
+            ));
+        }
+    }
```

Comment:
I thought about this. My argument is that if Bubba doesn't have fantastic control over the cursor (either due to his limitations or Hawkeye's), it would be useful to be able to make an animation more easily selectable.